schemas_extra.json

```diff
@@ -47,6 +47,15 @@
     "fileMatch": ["/package.json"],
     "url": "sublime://schemas/package-json-eslint"
   },
+  {
+    "fileMatch": [
+      "Preferences.sublime-settings",
+      "Preferences%20%28Linux%29.sublime-settings",
+      "Preferences%20%28OSX%29.sublime-settings",
+      "Preferences%20%28Windows%29.sublime-settings"
```

Comment:
It might be a bug that it requires parenthesis and space to be URL encoded. I had to debug the server to figure out how to make those work.